samples/subsys/ipc/openamp_rsc_table/README.rst

```diff
@@ -0,0 +1,42 @@
+.. _openAMP_rsc_table_sample:
+
+OpenAMP Sample Application using resource table
+###############################################
+
+Overview
+********
+
+This application demonstrates how to use OpenAMP with Zephyr based on a resource
+table. It is designed to respond to the Linux rpmsg client sample.
+This integration is expected platform independent.
+
+Building the application
+*************************
+
+.. zephyr-app-commands::
+   :zephyr-app: samples/subsys/ipc/openamp_linux_responder
+   :goals: test
+
+Open a serial terminal (minicom, putty, etc.) and connect the board with the
+following settings:
+
+- Speed: 115200
+- Data: 8 bits
+- Parity: None
+- Stop bits: 1
+
+Reset the board and the following message will appear on the corresponding
```

Comment:
Before you reset the board, don't you need to run the referenced Linux rpmsg client sample somewhere?